index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <main id="main">
+      <h1 id="title">This is Zakariya & Usama Tribute Page</h1>
+      <div id="img-div">
+        <img class="center" id="image" src="#" alt="Zack&Usama">
```

Comment:
Kindly add a link to the image as it was a requirement of the project. With this, I think it would be better to use a meaningful alt attribute.